library/src/main/java/com/alamkanak/weekview/DateTimeInterpreter.java

```diff
@@ -7,5 +7,5 @@
  */
 public interface DateTimeInterpreter {
     String interpretDate(Calendar date);
-    String interpretTime(int hour);
+    String interpretTime(int hour, int minutes);
```

Comment:
Hold on, that doesn't actually help. Just by adding the interpretTime(int hour, int minutes) to the interface will break an client implementing that interface. Only if we had a base class and people extended that, could we then add a method to the interface and not break consumer.